test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -429,20 +441,29 @@ class GrpclbEnd2endTest : public ::testing::Test {
     balancers_.at(i)->add_response(response, delay_ms);
   }
 
-  std::vector<std::pair<Status, EchoResponse>> SendRpc(const string& message,
-                                                       int num_rpcs,
-                                                       int timeout_ms = 1000) {
-    std::vector<std::pair<Status, EchoResponse>> results;
+  Status SendRpc(EchoResponse* response = nullptr) {
+    const bool local_response = (response == nullptr);
+    if (local_response) response = new EchoResponse;
     EchoRequest request;
-    EchoResponse response;
-    request.set_message(message);
-    for (int i = 0; i < num_rpcs; i++) {
-      ClientContext context;
-      context.set_deadline(grpc_timeout_milliseconds_to_deadline(timeout_ms));
-      Status status = stub_->Echo(&context, request, &response);
-      results.push_back(std::make_pair(status, response));
+    request.set_message(kRequestMessage_);
+    ClientContext context;
+    Status status = stub_->Echo(&context, request, response);
+    if (local_response) delete response;
+    return status;
+  }
+
+  void CheckRpcSendOk(const size_t times = 1) {
+    for (size_t i = 0; i < times; ++i) {
+      EchoResponse response;
+      const Status status = SendRpc(&response);
+      EXPECT_TRUE(status.ok());
```

Comment:
Suggest logging the code and message if this expectation fails, as was done in the original code.